SPECS/groff/groff.spec

```diff
@@ -42,15 +46,17 @@ rm -rf %{buildroot}%{_infodir}
 %{_defaultdocdir}/%{name}-%{version}/*
 %{_datarootdir}/%{name}/*
 %{_mandir}/*/*
-%changelog
-* Sat May 09 00:21:28 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.22.3-4
-- Added %%license line automatically
 
+%changelog
+*   Mon Sep 28 2020 Daniel McIlvaney <damcilva@microsoft.com> - 1.22.3-5
```

Comment:
There shouldn't be `-` before the version. (Same applies to Nick's entry).

Reply:
Fixed

Reply:
Fixed